scripts/Inelastic/IndirectReductionCommon.py

```diff
@@ -115,7 +119,7 @@ def load_files(data_files, ipf_filename, spec_min, spec_max, sum_files=False, lo
 
     logger.information('Summed workspace names: %s' % (str(workspace_names)))
 
-    return workspace_names, chopped_data
+    return workspace_names, chopped_data, masked_detectors
```

Comment:
only real comment here, I'm not sure I like having a dummy variable in all other uses of this function other than one, in python im fairly certain its possible to have the return type itself vary, so you could use 
```
if find_masked_detectors:
    masked_detectors = get_detectors_to_mask_from_groups(workspace_names)
    return workspace_names, chopped_data, masked_detectors
return workspace_names, chopped_data
```

that way the dummy variable is unnecessary when you call the function with `find_masked_detectors=false`
and you avoid the need for a dummy variable the majority of the time the function is used.